gradle.properties

```
@@ -0,0 +1,24 @@
+version=4.0.4
+#
+# main dependencies
+#
+hivemq-extension-sdk.version=4.0.0
+simpleclient-dropwizard.version=0.9.0
+simpleclient-servlet.version = 0.9.0
+simpleclient-common.version= 0.9.0
```

please use one version
```suggestion
prometheus-simpleclient.version=0.9.0
```